src/components/Popover/Popover.tsx

```diff
@@ -141,7 +152,7 @@ const Popover: FC<Props> = (props: Props) => {
       delay={delay}
       plugins={addTippyPlugins(hideOnEsc, addBackdrop)}
       // add arrow height to default offset if arrow is shown:
-      offset={[0, showArrow ? ARROW_HEIGHT + OFFSET : OFFSET]}
+      offset={[distance, skidding]}
```

Comment:
this should be 
```suggestion
      offset={[skidding, distance]}
```
see in the docs: https://atomiks.github.io/tippyjs/v6/all-props/#offset